Autofish crashes MCC on 1.16.2 version

**Bug description**
Autofish crashes MCC on 1.16.2 version.

```
>/useitem
[MCC] Used an item
>
Unhandled Exception:
System.IO.InvalidDataException: Failed to process incoming packet of type SpawnEntity. (PacketID: 0, Protocol: 751, LoginPhase: False, InnerException: System.IO.InvalidDataException). ---> System.IO.InvalidDataException: Unknown Entity ID 107. Is Entity Palette up to date for this Minecraft version?
  at MinecraftClient.Mapping.EntityPalettes.EntityPalette.FromId (System.Int32 id, System.Boolean living) [0x00050] in <123f70f18695447b96d7692986c22761>:0 
  at MinecraftClient.Protocol.Handlers.DataTypes.ReadNextEntity (System.Collections.Generic.Queue`1[T] cache, MinecraftClient.Mapping.EntityPalettes.EntityPalette entityPalette, System.Boolean living) [0x00034] in <123f70f18695447b96d7692986c22761>:0 
  at MinecraftClient.Protocol.Handlers.Protocol18Handler.HandlePacket (System.Int32 packetID, System.Collections.Generic.Queue`1[T] packetData) [0x0142d] in <123f70f18695447b96d7692986c22761>:0 
   --- End of inner exception stack trace ---
  at MinecraftClient.Protocol.Handlers.Protocol18Handler.HandlePacket (System.Int32 packetID, System.Collections.Generic.Queue`1[T] packetData) [0x01d75] in <123f70f18695447b96d7692986c22761>:0 
  at MinecraftClient.Protocol.Handlers.Protocol18Handler.Update () [0x00036] in <123f70f18695447b96d7692986c22761>:0 
  at MinecraftClient.Protocol.Handlers.Protocol18Handler.Updater () [0x00007] in <123f70f18695447b96d7692986c22761>:0 
  at System.Threading.ThreadHelper.ThreadStart_Context (System.Object state) [0x00014] in <9f0df102fe6e4cfea29d2e46f585d8a5>:0 
  at System.Threading.ExecutionContext.RunInternal (System.Threading.ExecutionContext executionContext, System.Threading.ContextCallback callback, System.Object state, System.Boolean preserveSyncCtx) [0x00071] in <9f0df102fe6e4cfea29d2e46f585d8a5>:0 
  at System.Threading.ExecutionContext.Run (System.Threading.ExecutionContext executionContext, System.Threading.ContextCallback callback, System.Object state, System.Boolean preserveSyncCtx) [0x00000] in <9f0df102fe6e4cfea29d2e46f585d8a5>:0 
  at System.Threading.ExecutionContext.Run (System.Threading.ExecutionContext executionContext, System.Threading.ContextCallback callback, System.Object state) [0x0002b] in <9f0df102fe6e4cfea29d2e46f585d8a5>:0 
  at System.Threading.ThreadHelper.ThreadStart () [0x00008] in <9f0df102fe6e4cfea29d2e46f585d8a5>:0 
[ERROR] FATAL UNHANDLED EXCEPTION: System.IO.InvalidDataException: Failed to process incoming packet of type SpawnEntity. (PacketID: 0, Protocol: 751, LoginPhase: False, InnerException: System.IO.InvalidDataException). ---> System.IO.InvalidDataException: Unknown Entity ID 107. Is Entity Palette up to date for this Minecraft version?
  at MinecraftClient.Mapping.EntityPalettes.EntityPalette.FromId (System.Int32 id, System.Boolean living) [0x00050] in <123f70f18695447b96d7692986c22761>:0 
  at MinecraftClient.Protocol.Handlers.DataTypes.ReadNextEntity (System.Collections.Generic.Queue`1[T] cache, MinecraftClient.Mapping.EntityPalettes.EntityPalette entityPalette, System.Boolean living) [0x00034] in <123f70f18695447b96d7692986c22761>:0 
  at MinecraftClient.Protocol.Handlers.Protocol18Handler.HandlePacket (System.Int32 packetID, System.Collections.Generic.Queue`1[T] packetData) [0x0142d] in <123f70f18695447b96d7692986c22761>:0 
   --- End of inner exception stack trace ---
  at MinecraftClient.Protocol.Handlers.Protocol18Handler.HandlePacket (System.Int32 packetID, System.Collections.Generic.Queue`1[T] packetData) [0x01d75] in <123f70f18695447b96d7692986c22761>:0 
  at MinecraftClient.Protocol.Handlers.Protocol18Handler.Update () [0x00036] in <123f70f18695447b96d7692986c22761>:0 
  at MinecraftClient.Protocol.Handlers.Protocol18Handler.Updater () [0x00007] in <123f70f18695447b96d7692986c22761>:0 
  at System.Threading.ThreadHelper.ThreadStart_Context (System.Object state) [0x00014] in <9f0df102fe6e4cfea29d2e46f585d8a5>:0 
  at System.Threading.ExecutionContext.RunInternal (System.Threading.ExecutionContext executionContext, System.Threading.ContextCallback callback, System.Object state, System.Boolean preserveSyncCtx) [0x00071] in <9f0df102fe6e4cfea29d2e46f585d8a5>:0 
  at System.Threading.ExecutionContext.Run (System.Threading.ExecutionContext executionContext, System.Threading.ContextCallback callback, System.Object state, System.Boolean preserveSyncCtx) [0x00000] in <9f0df102fe6e4cfea29d2e46f585d8a5>:0 
  at System.Threading.ExecutionContext.Run (System.Threading.ExecutionContext executionContext, System.Threading.ContextCallback callback, System.Object state) [0x0002b] in <9f0df102fe6e4cfea29d2e46f585d8a5>:0 
  at System.Threading.ThreadHelper.ThreadStart () [0x00008] in <9f0df102fe6e4cfea29d2e46f585d8a5>:0 
```

--- a/MinecraftClient/Protocol/Handlers/Protocol18.cs
+++ b/MinecraftClient/Protocol/Handlers/Protocol18.cs
@@ -115,7 +115,7 @@ namespace MinecraftClient.Protocol.Handlers
             {
                 if (protocolversion > MC1163Version && handler.GetEntityHandlingEnabled())
                     throw new NotImplementedException("Please update entity types handling for this Minecraft version. See EntityType.cs");
-                if (protocolversion >= MC1163Version)
+                if (protocolversion >= MC1162Version)
                     entityPalette = new EntityPalette1162();
                 else if (protocolversion >= MC116Version)
                     entityPalette = new EntityPalette1161();
@@ -132,7 +132,7 @@ namespace MinecraftClient.Protocol.Handlers
             {
                 if (protocolversion > MC1163Version && handler.GetInventoryEnabled())
                     throw new NotImplementedException("Please update item types handling for this Minecraft version. See ItemType.cs");
-                if (protocolversion >= MC1163Version)
+                if (protocolversion >= MC1162Version)
                     itemPalette = new ItemPalette1162();
                 else itemPalette = new ItemPalette1161();
             }

